src/textureremix/TextureRemix.java

```diff
@@ -48,6 +53,24 @@ of this software and associated documentation files (the "Software"), to deal
      * @param args the command line arguments
      */
     public static void main(String[] args) {
+        // Check for savepath persistence file
+        File f = new File( "savepath.txt" );
```

Comment:
I like this, but could you put it in a separate function? For the sake of keeping the main function tidy.